app/views/sessions/new.html.haml

```diff
@@ -6,3 +6,4 @@
         = f.text_field :email, placeholder: "Email"
         = f.password_field :password, placeholder: "Password"
         = f.submit "Login"
+        = link_to('Forgot Password', new_password_reset_path)
```

Comment:
Yea, it is just a link right now, but I think that might be pretty common actually